android/library/src/main/java/com/polidea/multiplatformbleadapter/AdvertisementData.java

```diff
@@ -44,6 +44,22 @@ public Integer getTxPowerLevel() {
         return solicitedServiceUUIDs;
     }
 
+    private AdvertisementData() {}
+
+    public AdvertisementData(byte[] manufacturerData,
+                             Map<UUID, byte[]> serviceData,
+                             ArrayList<UUID> serviceUUIDs,
```

Comment:
I suppose the type should be `List` instead of `ArrayList`

Reply:
It can't be without changing the type of fields in this class. Should I do that?